public/cms.html

```diff
@@ -13,92 +13,74 @@
-		
-		  </ul>
-		
-		<div class="row white-text">
-			<div class="col-md-6 offset-md-3">
-				<form id="cms">
-					<div class="form-group">
-						<label for="title">Title:</label>
-						<input placeholder="Post Title" type="text" class="form-control" id="title">
-						<br />
-						<label for="body">Text:</label>
-						<textarea placeholder="Text" class="form-control" rows="10" id="body"></textarea>
-						<div class="form-group">
-							<label for="author">Author:</label>
-							<select class="custom-select" id="author">
-								<option value="" disabled selected>Choose author option</option>
-							</select>
-						</div>
-						<!-- Added category field -->
 						<div class="form-group">
-							<label for="category">Category:</label>
-							<select class="custom-select" id="category">
-								<option value="" disabled selected>Choose categorie option</option>
-								<option value="cooking">Cooking</option>
-								<option value="DIY">DIY</option>
-								<option value="Travel">Travel</option>
-								<option value="Soccer">Soccer</option>
-								<option value="Inspirational Quotes">Inspirational Quotes</option>
-								<option value="Music">Music</option>
-							</select>
+							<label for="title">Title:</label>
+							<input placeholder="Post Title" type="text" class="form-control" id="title">
+							<br />
+							<label for="body">Text:</label>
+							<textarea placeholder="Text" class="form-control" rows="10" id="text"></textarea>
+							<div class="form-group">
+								<label for="author">Author:</label>
+								<select class="custom-select" id="author">
+									<option value="" disabled selected>Choose author option</option>
+								</select>
+							</div>
+							<!-- Added category field -->
+							<div class="form-group">
+								<label for="category">Category:</label>
+								<select class="custom-select" id="category">
+									<option value="" disabled selected>Choose categorie option</option>
+									<option value="cooking">Cooking</option>
+									<option value="DIY">DIY</option>
+									<option value="Travel">Travel</option>
+									<option value="Soccer">Soccer</option>
+									<option value="Inspirational Quotes">Inspirational Quotes</option>
+									<option value="Music">Music</option>
+								</select>
+							</div>
+
+							<br />
+							<br />
+							<button type="submit" class="btn btn-success submit">Submit</button>
 						</div>
-				
-						<br />
-						<br />
-						<button type="submit" class="btn btn-success submit">Submit</button>
-					</div>
-				</form>
+					</form>
+				</div>
 			</div>
 		</div>
-	</div>
-	<!-- Custom Script -->
+</main>
 
+<footer class="page-footer transparent center-align">
+	<div class="container">
+		<div class="row">
+			<div>
+				<h5 class="white-text">Casper and Company</h5>
+				<p>Practicing social distancing since © 2020 </p>
+			</div>
+		</div>
+	</div>
+</footer>
 
 
-  <!-- Compiled and minified JavaScript -->
```

Comment:
add this all back in `  <!-- Compiled and minified JavaScript -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
  <script src="./js/api.js"></script> 
  <script src="./js/author.js"></script>
  <script src="./js/index.js"></script>
  <script src="./js/blog.js"></script>
  <script src="./js/cms.js"></script>
	<script>
		$(document).ready(function () {
			$('select').formSelect();
		});

	</script>`